test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorViewEngineOptionsTest.cs

```diff
@@ -12,6 +13,65 @@ namespace Microsoft.AspNetCore.Mvc.Razor
     public class RazorViewEngineOptionsTest
     {
         [Fact]
+        public void AreaViewLocationFormats_ContainsExpectedLocations()
+        {
+            // Arrange
+            var services = new ServiceCollection().AddOptions();
+            var areaViewLocations = new[]
+            {
+                "/Areas/{2}/MvcViews/{1}/{0}.cshtml",
+                "/Areas/{2}/MvcViews/Shared/{0}.cshtml",
+                "/MvcViews/Shared/{0}.cshtml"
+            }.AsEnumerable();
```

Comment:
Throw `AsEnumerable()` into the nearest garbage heap. Not sure why the `Equal` assertion isn't working correctly with an array and a list but just `new`ing up a `List<string>` would be a much better workaround.